dart/lib/src/protocol/sentry_id.dart

```diff
@@ -0,0 +1,16 @@
+/// Sentry response id
+
+const _dashIndexed = [8, 13, 18, 23];
+
+class SentryId {
+  static final String emptyId =
+      List.generate(36, (index) => _dashIndexed.contains(index) ? '-' : '0')
+          .join();
+
+  /// The ID Sentry.io assigned to the submitted event for future reference.
+  final String id;
```

Comment:
Could we have an override on toString instead? Ideally we'd use an instance hereas a string (because it really is)